app/controllers/api/v2/subnets_controller.rb

```diff
@@ -89,6 +89,17 @@ def destroy
       def freeip
       end
 
+      api :GET, "/subnets/:proxy_id/import_ipv4_subnets", N_("Import Ipv4 subnet from Smart proxy")
```

Comment:
What are your considerations for making this explicitly IPv4-only? While our DHCP API currently only supports IPv4, it's not explicit anywhere. I'd prefer to have it just
```suggestion
      api :GET, "/subnets/:proxy_id/import", N_("Import subnets from the Smart Proxy")
```

However, reading that I think it's still not right. Everywhere `/subnets/:id` is the subnet ID, not the Smart Proxy. Having it mixed is very confusing.

I think it should really be something like `/api/smart_proxies/:id/import_subnets`.

Reply:
@ekohl  Thanks for your comment, this is what I thought as well but OLeh said otherwise here: https://community.theforeman.org/t/import-ipv4-subnet-features-smart-proxy/24865
regarding the Ipv4, I wanted to be consistent with the UI.

Reply:
I don't see what @ofedoren said about the path in that thread. Perhaps he can enlighten us here.

Reply:
he wrote that the option import-ipv4-subnets doesn't appear in hammer subnet --help.

Reply:
Well, that wasn't me, but the reporter of the feature request :) And if we want to be consistent with the UI then the new command should indeed be under smart-proxy subcommand and thus be related to the proxy controller as @ekohl says, not the subnets.